RPS.html

```diff
@@ -67,27 +67,61 @@
       }
     }
 
+    let playerScore = 0;
+    let computerScore = 0;
+
     function game(playerSelection) {
       const computerSelection = computerPlay();
 
       const resultGameDiv = document.querySelector('.result');
-      const pRoundRPS = document.createElement('p');
-      const pUserSelect = document.createElement('p');
-      const pCompSelect = document.createElement('p');
+      const roundResultParagraph = document.createElement('p');
+      const userChoiceParagraph = document.createElement('p');
+      const computerChoiceParagraph = document.createElement('p');
+      const playerScoreParagraph = document.createElement('p');
+      const computerScoreParagraph = document.createElement('p');
+      
+      const playerRockWin = "You won! Rock beats Scissors.";
+      const playerRockLose = "You lose! Rock beats Scissors.";
+      const playerPaperWin = "You won! Paper beats rock.";
+      const playerPaperLose = "You lose! Paper beats rock.";
+      const playerScissorsWin = "You won! Scissors beats paper.";
+      const playerScissorsLose = "You lose! Scissors beats paper";
+
+      let roundResult = playRound(playerSelection, computerSelection);
+
+      if (roundResult === playerRockWin || roundResult === playerPaperWin || roundResult === playerScissorsWin) {
+        playerScore += 1;
+      } else if (roundResult === playerRockLose || roundResult === playerPaperLose || roundResult === playerScissorsLose) {
+        computerScore += 1;
+      }
```

Comment:
Ok, let's refactor this. 
1. For the first if condition, we can use array's `some` in this case. https://learn.javascript.ru/array-iteration#every-some
- put winning strings in an array, like `playerRockWin`
- use `some` on the array to find out if any of the values in it that array equals `roundResult`

2. After the first if condition, we only need to check for draw, there is no need to check everything all over again. So replace the second if condition with a draw check.